runtime/gc_vlhgc/GlobalMarkingScheme.cpp

```diff
@@ -792,7 +792,7 @@ void
 MM_GlobalMarkingScheme::scanContinuationNativeSlots(MM_EnvironmentVLHGC *env, J9Object *objectPtr, ScanReason reason)
 {
 	J9VMThread *currentThread = (J9VMThread *)env->getLanguageVMThread();
-	if (MM_GCExtensions::needScanStacksForContinuationObject(currentThread, objectPtr)) {
+	if (MM_GCExtensions::needScanStacksForContinuationObject(currentThread, objectPtr, true)) {
```

Comment:
let's introduce a line 
const bool isGlobalGC = true;
and instead of passing 'true' pass isGlobalGC at the 2 spots
that will help with undestanding with what these flags we pass are about

update it for all other methods that use GC_VMThreadStackSlotIterator::scanSlots()